docs/spfx/build-for-teams-configure-in-teams.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Configure SharePoint Framework web parts in Microsoft Teams
+description: To accommodate your users' preferences, you can let them configure your web parts when used in Microsoft Teams.
+ms.date: 06/15/2020
+ms.prod: sharepoint
+localization_priority: Normal
+---
+
+# Configure SharePoint Framework web parts in Microsoft Teams
+
+To accommodate your users' preferences, you can let them configure your web parts when used in Microsoft Teams. Depending if you exposed your web parts as Teams tabs or personal apps, there are different ways to implement configuration capabilities in your web part.
+
+> [!TIP]
+> To see how to use the different concepts described in this article, see the sample [Leads application](https://github.com/pnp/sp-dev-solutions/tree/master/solutions/LeadsLOBSolution) on GitHub.
+
+## Configure Microsoft Teams tabs built using SharePoint Framework
+
+Typically, when building Microsoft Teams tab, you need to [build custom UI to allow users to configure your tab](https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/create-tab-pages/configuration-page). Additionally, you need to write code to store and load configuration values as selected by the user.
+
+When building tabs using SharePoint Framework, the generated tab uses the [web part property pane](web-parts/guidance/integrate-web-part-properties-with-sharepoint.md) to let users configure the tab. This saves you a lot of effort. Not only you don’t need to build and maintain a separate configuration UI but you also don’t need to implement any code responsible for storing and managing the settings. All of that is handled automatically for you by SharePoint Framework.
+
+## Configure Microsoft Teams personal apps built using SharePoint Framework
+
+Microsoft Teams personal apps don’t offer any infrastructure for implementing configuration. Instead, following the pattern recommended by Microsoft Teams, personal app’s [settings should be exposed in a separate tab](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/design/personal-apps).
+
+:::image type="content" source="../images/build-for-teams/build-for-teams-personal-app.png" alt-text="Personal app with multiple tabs including settings built using SharePoint Framework":::
```

Comment:
The image format should be in the markdown syntax of `![alt image text](URL to image)`

Reply:
This is the syntax that's coming from the VSCode Docs Authoring Pack extension. If MD is preferred, I'll update all image references to that 👍